slider/beatmap.py

```diff
@@ -1680,6 +1683,31 @@ def _resolve_stacking_old(self, hit_objects, ar, cs):
 
         return hit_objects
 
+    def closest_hitobject(self, t):
```

Comment:
`t` should be a timedelta object so that the units are passed with the value. This makes is very difficult to misread or mess up the calls, especially when times are passed around a larger system. This also makes it easy to compare with the time on the hit object itself.